app/globals.css

```
@@ -597,6 +597,135 @@ button.word-card:focus-visible {
+    border-radius: 0;
+    background: transparent;
+  }
+
+  .rules-role-table thead {
+    position: absolute;
+    width: 1px;
+    height: 1px;
+    padding: 0;
+    margin: -1px;
+    overflow: hidden;
+    clip: rect(0, 0, 0, 0);
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)